packages/ui-core/core/src/lib/auth/cookie-helper.ts

```
@@ -0,0 +1,164 @@
+	// Prepare cookie options with defaults
+	const cookieOptions = {
+		path: '/',
+		SameSite: 'Lax',
+		Secure: window.location.protocol === 'https:',
+		...options
+	};
+
+	// Remove domain option for localhost or 127.0.0.1
+	const hostname = window.location.hostname;
+	if (hostname === 'localhost' || hostname === '127.0.0.1') {
+		delete cookieOptions['domain'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)